velox/common/memory/MemoryAllocator.h

```diff
@@ -347,6 +347,10 @@ class MemoryAllocator : public std::enable_shared_from_this<MemoryAllocator> {
 
   virtual std::string toString() const = 0;
 
+  static bool isPowerOfTwo(uint16_t size) {
```

Comment:
There is a helper in BitUtil.h

Reply:
That one looks quite terrible...Let me see if I can fix that one